### DISCUSSION TITLE
Does nginder support ignore SSL？

### DISCUSSION THREAD

**Discussion by Chen-kaiyuan** [naver/ngrinder#762] [Q&A] [answered]
Does nginder support ignore SSL？
I try to increase the parameter run java -Dhttps.protocols=TLSv1.2,TLSv1.1, is ok.
![image](https://user-images.githubusercontent.com/17740259/108617422-24974b00-7451-11eb-8597-9708f2700101.png)
Is there any other way?

**Comment by donggyu04** [comment, marked as answer]
@Chen-kaiyuan

Hi, try modify `agent.java_opt` option in `agent.conf` like below.
If it doesn't work, currently there is no way.

```
// agent.conf
...

 # provide more agent java execution option if necessary.
 agent.java_opt=-Dhttps.protocols=TLSv1.1,TLSv1.2
 # provide more agent jvm classpath if necessary.

...
```

> agent.conf is located in under AGENT_HOME that commonly {USER_HOME}/.ngrinder_agent

**Reply by Chen-kaiyuan** [reply]
Thanks♪(･ω･)ﾉ
